data/pathfinder/dreamscarred_press/path_of_war/pow_abilitycategories_psi.lst

```diff
@@ -3,7 +3,7 @@
 # Original Entry Date: 2015-06-11
 
 # Ability Category						Visible		Editable?		Change Pool?	Fractional values?	Category of Object		Type of Object				Plural description for UI			Display Location
-ABILITYCATEGORY:Pathwalker Discipline			VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES	FRACTIONALPOOL:NO		CATEGORY:Internal			TYPE:PathwalkerAbilityChoice		PLURAL:Pathwalker Disciplines			DISPLAYLOCATION:Class Features
+ABILITYCATEGORY:Pathwalker Discipline			VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES	FRACTIONALPOOL:NO		CATEGORY:Internal			TYPE:PathwalkerDisciplineChoice		PLURAL:Pathwalker Disciplines			DISPLAYLOCATION:Class Features
```

Comment:
This was probably the only change you needed to make here. The other is superficial, since everything in those two abilities are being loaded anyways.